builder/create.sh

```diff
@@ -51,12 +51,6 @@ do
     mv $file ${file/ServiceName/$SVC}
 done
 
-echo "regenerating proto buff"
-p=`pwd`
-cd ${SVC}/${SVC}_proto/
-bash generate.sh
-cd $p
-
 echo "creating $SVC project"
```

Comment:
we need to remove Orion/builder from file content to be able to generate proto properly